code/controllers/subsystem/objects.dm

```diff
@@ -17,14 +17,12 @@ var/datum/subsystem/objects/SSobj
 
 /datum/subsystem/objects/Initialize(timeofday, zlevel)
 	setupGenetics()
-	for(var/atom/movable/AM in world)
-		if (zlevel && AM.z != zlevel)
+	for(var/atom/A in world)
```

Comment:
Does this incur any significant slowdown? Also, everything in `world` is guaranteed to be an atom, so you can just
```
for(var/V in world)
	var/atom/A = V
```
which is faster